frontend/src/store/auth/actions.ts

```diff
@@ -39,5 +40,13 @@ const loadCurrentUser = createAsyncThunk<
   const { user } = await authApi.getCurrentUser();
   return user;
 });
+const logOut = createAsyncThunk<void, NavigateFunction, AsyncThunkConfig>(
+  ActionType.LOG_OUT,
+  async (navigate, { extra }) => {
+    const { storage } = extra;
+    storage.removeItem(StorageKey.TOKEN);
+    navigate(AppRoute.SIGN_IN);
```

Comment:
```suggestion
    navigate(AppRoute.SIGN_IN);
```
use navigation service (from `extra`)